cmake/configs/default.cmake

```diff
@@ -184,6 +184,7 @@ hunter_config(xrender VERSION 0.9.7)
 hunter_config(xshmfence VERSION 1.2)
 hunter_config(xtrans VERSION 1.2.7)
 hunter_config(xxf86vm VERSION 1.1.2)
+hunter_config(YAML_CPP VERSION 0.5.3)
```

Comment:
if the package is to be named with uppercase, can this be moved with the upper case packages above the lowercase?